runtime/dart_isolate.h

```diff
@@ -232,7 +231,29 @@ class DartIsolate : public UIDartState {
       std::optional<std::string> dart_entrypoint,
       std::optional<std::string> dart_entrypoint_library,
       std::unique_ptr<IsolateConfiguration> isolate_configration,
-      std::shared_ptr<VolatilePathTracker> volatile_path_tracker);
+      std::shared_ptr<VolatilePathTracker> volatile_path_tracker,
+      const DartIsolate* spawning_isolate = nullptr);
+
+  //----------------------------------------------------------------------------
+  /// @brief     Creates a running DartIsolate who shares as many resources as
+  ///            possible with the caller DartIsolate.  This allows them to
+  ///            occupy less memory together.
+  /// @attention Only certain setups can take advantage of the most savings
+  ///            currently, release builds currently.
```

Comment:
@mkustermann where do you estimate you guys will land by the next Flutter stable?